### DISCUSSION TITLE
Add a new Assembly Option "Draw arrows on right" below the "Draw arrows for jumps" Assembly Option

### DISCUSSION THREAD

**Discussion by ghost** [grumpycoders/pcsx-redux#1105] [Ideas]
By default PCSX-Redux draws all jumps arrows on the **left** side of the Assembly window or tab unlike PCSX2 whose debugger draws these jumps arrows on the **right** side of the assembly view but with this new Assembly Option the user can choose to draw jumps arrows on the **right** side of the Assembly window or tab instead with PCSX-Redux for convenience purposes.

Just another quality of service feature.

**Comment by wheremyfoodat** [comment]
Also probably not happening. Not much point in styling the GUI the exact same way as PCSX2, and at some point having too much customizability hurts UX.

**Reply by ghost** [reply]
> Also probably not happening. Not much point in styling the GUI the exact same way as PCSX2, and at some point having too much customizability hurts UX.

I did**n't** mean to change the GUI of PCSX-Redux to be like PCSX2 at all!

This is just an idea to add a new option for users who want to see arrows come out from the right side of the branch instructions instead of left because on PCSX2 the arrows come out from the right side of the branch instructions and not left. That's all but if this is too complicated and too much work for you then you can give this feature request **lowest** priority then.
